pset-functions.js

```diff
@@ -174,7 +214,12 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
+function isPalindrome(word, i = 0) {
+  const j = word.length - 1 - i;
+  return i >= j ? true : word[i] === word[j] && isPalindrome(word, i + 1);
+}
 
+console.log(isPalindrome('run'))
 // Add your code above this line
```

Comment:
@OcielC Good work here on the solution. I see you are using recursion here in this solution, nice way to solve the problem.

Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L242)